js/views/reader_view.js

```diff
@@ -244,6 +246,8 @@ var ReaderView = function (options) {
 
             _internalLinksSupport.processLinkElements($iframe, spineItem);
 
+            _externalAgentSupport.bindToContentDocument($iframe[0].contentDocument, spineItem);
+
```

Comment:
Just nit-picking (code aesthetics): this could be moved below `var contentDoc = $iframe[0].contentDocument;` (to reuse the `contentDoc` var).